package/d-installer-cli.spec

```diff
@@ -0,0 +1,65 @@
+#
+# spec file for package d-installer-cli
+#
+# Copyright (c) 2023 SUSE LINUX GmbH, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via http://bugs.opensuse.org/
+#
+
+Name:           d-installer-cli
+#               This will be set by osc services, that will run after this.
+Version:        0
+Release:        0
+Summary:        D-Installer command line interface
+#               If you know the license, put it's SPDX string here.
+#               Alternately, you can use cargo lock2rpmprovides to help generate this.
+License:        GPL-2.0-only
+Url:            https://github.com/yast/d-installer-cli
+Source0:        %{name}-%{version}.tar.zst
+Source1:        vendor.tar.zst
+Source2:        cargo_config
```

Comment:
I am missing in package file this cargo_config...or is it generated automatic?

Reply:
Good point. It is generated by the [cargo_vendor service](https://github.com/openSUSE/obs-service-cargo_vendor), but I will add a note to clarify it.